src/core/state.go

```diff
@@ -50,6 +50,18 @@ type Task struct {
 	Run   uint32 // Only present for tests (the run of a build is always zero)
 }
 
+// A OutputDownloadOption is the option for how outputs should be downloaded.
+type OutputDownloadOption uint8
+
+const (
```

Comment:
Should we use iota here? Or does it not matter because it's not very many constants?

Reply:
I guess iota is better.